types/mini-css-extract-plugin/index.d.ts

```diff
@@ -6,18 +6,25 @@
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { ChunkData, Plugin } from 'webpack';
+import { Configuration, Compiler } from 'webpack';
 
 /**
  * Lightweight CSS extraction webpack plugin
  * This plugin extract CSS into separate files. It creates a CSS file per JS file which contains CSS. It supports On-Demand-Loading of CSS and SourceMaps.
  * Configuration Detail: https://github.com/webpack-contrib/mini-css-extract-plugin#configuration
  */
-declare class MiniCssExtractPlugin extends Plugin {
-    /** webpack loader used always at the end of loaders list */
+declare class MiniCssExtractPlugin {
+    /**
+     * Webpack loader used always at the end of loaders list
+     */
     static loader: string;
 
     constructor(options?: MiniCssExtractPlugin.PluginOptions);
+
+    /**
+     * Apply the plugin
+     */
+    apply(compiler: Compiler): void;
```

Comment:
Wondering why there is no `Plugin` interface any more.

Reply:
There is, it's backported and exported as `WebpackPluginInstance` contract, compatible with 4.*/5.*,
but there is not strict requirement really to implement the interface here.